src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -221,6 +222,33 @@ private void SolutionManager_ActionsExecuted(object sender, ActionsExecutedEvent
             }
         }
 
+        private void SolutionManager_CacheUpdated(object sender, NuGetEventArgs<string> e)
+        {
+            if (Model.IsSolution)
+            {
+                // This means that the UI is open for the solution.
+                Refresh();
+            }
+            else
+            {
+                // This is a project package manager, so there is one and only one project.
+                var project = Model.Context.Projects.First();
```

Comment:
discussed offline. Keeping this since it is the same as the code for `SolutionManager_ActionsExecuted`.